assets/dev/scss/frontend/widgets/playing-cards.scss

```diff
@@ -0,0 +1,43 @@
+.elementor-playing-cards-wrapper {
```

Comment:
`.elementor-widget-playing cards`. 

We always define the css variables with the main widget wrapper. See e.g. modules/nested-tabs/assets/scss/frontend.scss